datadog_checks_dev/datadog_checks/dev/tooling/configuration/consumers/example.py

```diff
@@ -183,7 +183,11 @@ def write_option(option, writer, indent='', start_list=False):
             multiple = option['multiple']
             multiple_instances_defined = option.get('multiple_instances_defined')
 
-            writer.write(indent, option_name, ':', '\n')
+            if not option_enabled(option):
```

Comment:
shouldn't it be `true` by default?

Reply:
It seems for the other non-nested options that if the option is not explicitly `enabled`, it is commented out: [options with a `value` option](https://github.com/DataDog/integrations-core/pull/11707/files/ee0d7193ac81ba8ef29714028711d2720b3a588d#diff-4a6fd41fceca6e1697dad3df039b50fd490dcf793c3a70bb4bdf42352f1bcb87R170-R171) and sections with an [`example` option](https://github.com/DataDog/integrations-core/pull/11707/files/ee0d7193ac81ba8ef29714028711d2720b3a588d#diff-4a6fd41fceca6e1697dad3df039b50fd490dcf793c3a70bb4bdf42352f1bcb87R204-R205)